Jenkinsfile.gerrit

```diff
@@ -39,17 +40,17 @@ pipeline {
             parallel {
                 stage('Lint') {
                     steps {
-                        sh 'go run ./scripts/check-copyright.go'
-                        sh 'go run ./scripts/check-large-files.go'
-                        sh 'go run ./scripts/check-imports.go -race ./...'
-                        sh 'go run ./scripts/check-peer-constraints.go -race -fail=false'
-                        sh 'go run ./scripts/protobuf.go --protoc=$HOME/protoc/bin/protoc lint'
-                        sh 'go run ./scripts/protobuf.go --protoc=$HOME/protoc/bin/protoc check-lock'
-                        sh 'go run ./scripts/atomicalign.go ./...'
-                        sh 'go run ./scripts/check-errs.go ./...'
+                        sh 'cd ./scripts; go run ./check-copyright'
+                        sh 'cd ./scripts; go run ./check-large-files'
```

Comment:
These both above start at work dir, so by cd-ing into the folder it messes up things.

But `GOMOD=scripts/go.mod` could work, to avoid needing to be in scripts dir.

Reply:
Hmm, I got it to work somehow without needing to cd into the directory... I mean without having to change the tools. I'll try to remember what I did.

Reply:
Anyways, let's make the scripts take the path as an argument rather than hardcoding the "." in that case. Maybe it makes it more obvious what's happening.

I'll see on Monday whether I had a better solution.

Reply:
Apparently, no. My final version used `-mod=readonly` avoid modifying `go.mod`.